data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/heals/heals_affects_everybody.cfg

```diff
@@ -0,0 +1,87 @@
+#####
```

Comment:
A note on the filename - having a single file in this directory that starts `heals_` instead of `heal_` is annoying for tab-completion of filenames.

Reply:
Updated.

Reply:
Just noticed that the second `s`, on the end of `affects`, does the same. The 3 other tests are `heal_affect_...`

Reply:
Updated.